pkg/render/logstorage/esgateway/esgateway.go

```diff
@@ -119,12 +118,6 @@ func (e *esGateway) Objects() (toCreate, toDelete []client.Object) {
 	toCreate = append(toCreate, e.esGatewayServiceAccount())
 	toCreate = append(toCreate, e.esGatewayDeployment())
 
-	// The following secret is used by kube controllers and sent to managed clusters. It is also used by manifests in our docs.
```

Comment:
I think this is also still needed to support clusters <= v3.16 (that aren't connecting to Linseed)

Reply:
Ok I'll verify.